### PR TITLE
update login pages

### PR DESCRIPTION
**Pull request by NemTechCN**

### CODE REVIEW

File: src/core/model/AppWallet.ts [hunk 1]
```diff
@@ -555,18 +570,32 @@ export class AppWallet {
...
+                    accountPrivateKey,
+                    this.networkType,
+                    feeAmount,
+                );
         } catch (error) {
             throw new Error(error)
         }
     }
+
+    getTenAddressFromMnemonic(
```
Comment (evias) [inline src/core/model/AppWallet.ts]: what is this method used for?

File: src/router/routers.ts [hunk 2]
```diff
@@ -226,41 +226,76 @@ const routers = [
...
+                                name: 'step5',
+                                component: () => import('@/views/login/creat-account/step-5/Step5.vue'),
+                            }],
+                    }, {
+                        path: '/importAccount',
+                        name: 'importAccount',
+                        redirect: '/importAccount1',
+                        component: () => import('@/views/login/import-account/ImportAccount.vue'),
+                        children: [{
+                            path: '/importAccount1',
```
Comment (evias) [inline src/router/routers.ts]: why the numbered suffix?

File: src/router/routers.ts [hunk 3]
```diff
@@ -226,41 +226,76 @@ const routers = [
...
+                            name: 'importAccount1',
+                            component: () => import('@/views/login/import-account/import-account-1/ImportAccount1.vue'),
+                        }, {
+                            path: '/importAccount2',
```
Comment (evias) [inline src/router/routers.ts]: please use features names, not numbers

File: src/views/login/creat-account/CreatAccount.vue [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<template>
```
Comment (evias) [inline src/views/login/creat-account/CreatAccount.vue]: please do not add this component **with a typo in the name** and in the folder name..

File: src/views/login/creat-account/CreatAccountTs.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import {Vue, Component} from 'vue-property-decorator'
```
Comment (evias) [inline src/views/login/creat-account/CreatAccountTs.ts]: reproduced typo ... `CreateAccountTs`

File: src/views/login/creat-account/CreatAccountTs.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+import {step5} from '@/mock/login/index.ts'
+
+@Component
+export default class CreatAccountTs extends Vue {
```
Comment (evias) [inline src/views/login/creat-account/CreatAccountTs.ts]: reproduced typo ... `CreateAccountTs`

File: src/views/login/creat-account/step-1/Step1Ts.ts [hunk 7]
```diff
@@ -1,14 +1,28 @@
...
-export class CreateAccountTs extends Vue {
+@Component({
+    computed: {
+        ...mapState({
+            activeAccount: 'account',
+            app: 'app'
+        })
+    }
+})
+export class Step1Ts extends Vue {
```
Comment (evias) [inline src/views/login/creat-account/step-1/Step1Ts.ts]: name the step more pragmatically such that it helps understand what scope this step is used for.

example:

- Step 1 = `create name + select network`  => **`ImportStepCreateWallet`**

only use `step1` and `step2`, etc.. (numbered) in parent scope where you can map each page's component

File: src/views/login/creat-account/step-2/Step2Ts.ts [hunk 8]
```diff
@@ -0,0 +1,25 @@
+import {Vue, Component} from 'vue-property-decorator'
+
+@Component
+export default class Step2Ts extends Vue {
```
Comment (evias) [inline src/views/login/creat-account/step-2/Step2Ts.ts]: name the step more pragmatically such that it helps understand what scope this step is used for.

File: src/views/login/creat-account/step-3/Step3Ts.ts [hunk 9]
```diff
@@ -0,0 +1,20 @@
+import {Vue, Component} from 'vue-property-decorator'
+import {createMnemonic} from "@/core/utils"
+
+@Component
+export default class Step3Ts extends Vue {
```
Comment (evias) [inline src/views/login/creat-account/step-3/Step3Ts.ts]: name the step more pragmatically such that it helps understand what scope this step is used for.

File: src/views/login/creat-account/step-5/Step5Ts.ts [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+@Component({
+    computed: {
+        ...mapState({
+            activeAccount: 'account',
+            app: 'app'
+        })
+    }
+})
+export default class Step5Ts extends Vue {
```
Comment (evias) [inline src/views/login/creat-account/step-5/Step5Ts.ts]: name the step more pragmatically such that it helps understand what scope this step is used for.

File: src/core/model/AppWallet.ts (hunk 1 above)
Comment (NemTechCN) [reply src/core/model/AppWallet.ts]: this method used for generate 10 wallet with one seed, so the user can import 10 wallets while import a existed seed.

File: src/router/routers.ts (hunk 2 above)
Comment (NemTechCN) [reply src/router/routers.ts]: fixed

File: src/router/routers.ts (hunk 3 above)
Comment (NemTechCN) [reply src/router/routers.ts]: fixed

File: src/views/login/creat-account/CreatAccount.vue (hunk 4 above)
Comment (NemTechCN) [reply src/views/login/creat-account/CreatAccount.vue]: fixed

File: src/views/login/creat-account/CreatAccountTs.ts (hunk 5 above)
Comment (NemTechCN) [reply src/views/login/creat-account/CreatAccountTs.ts]: fixed

File: src/views/login/creat-account/CreatAccountTs.ts (hunk 6 above)
Comment (NemTechCN) [reply src/views/login/creat-account/CreatAccountTs.ts]: fixed

File: src/views/login/creat-account/step-1/Step1Ts.ts (hunk 7 above)
Comment (NemTechCN) [reply src/views/login/creat-account/step-1/Step1Ts.ts]: fixed

File: src/views/login/creat-account/step-2/Step2Ts.ts (hunk 8 above)
Comment (NemTechCN) [reply src/views/login/creat-account/step-2/Step2Ts.ts]: fixed

File: src/views/login/creat-account/step-3/Step3Ts.ts (hunk 9 above)
Comment (NemTechCN) [reply src/views/login/creat-account/step-3/Step3Ts.ts]: fixed

File: src/views/login/creat-account/step-5/Step5Ts.ts (hunk 10 above)
Comment (NemTechCN) [reply src/views/login/creat-account/step-5/Step5Ts.ts]: fixed

**Comment by NemTechCN** [comment]
> * (when localStorage empty) In login, you can click register, with any password
> * (when localStorage empty) The error `TypeError: Cannot read property 'accountName' of undefined` is always there
> * Run `npm install` and commit `package-lock.json`
> * It is not possible to scroll in `inputAccountInfo`, (I can't see the buttons on my screen)
> * Should be able to sumbit a form when pressing enter (can't in `inputAccountInfo`)
> * The verify backup mnemonic box can not contain all the words, and is not scrollable
> * create_successful_tips key is not in en-US
> * `SET_ACCOUNT_NAME` mutation is called but does not exist
> * in `importMnemonic` the green butons says "select the imported wallet public key", I think it is confusing
> * in the `walletChoose`, there is a balance column, but balances are not fetched
> * manage the images like in my PR, using the route meta object. Give meaningful names to the images and the import (createAccountStep1 is better than setp5_1)

fixed

**Comment by decentraliser** [comment]
Account import bug:

- Start with emtpy localStorage
- click Import seed
- fill the form, click "generating mnemonic"
- click "return to password setting"
- fill the form, with same account name as in 3)
- can not submit the form "Acount name already exists"
-

**Comment by decentraliser** [comment]
in importMnemonic, the buttons are not on the same row 
image

**Comment by decentraliser** [comment]
I'm not sure about the HD derivation path, I think it is confusing as is (I guess we can see that later)

image

**Comment by decentraliser** [comment]
If I set `_ENABLE_TREZOR_ = true` in the localStorage, I can't access to the Trezor account creation

**Comment by decentraliser** [comment]
in walletChoosen the balance column is broken
image

**Comment by decentraliser** [comment]
> Account import bug:
> 
> * Start with emtpy localStorage
> * click Import seed
> * fill the form, click "generating mnemonic"
> * click "return to password setting"
> * fill the form, with same account name as in 3)
> * can not submit the form "Acount name already exists"

fixed

**Comment by decentraliser** [comment]
> in importMnemonic, the buttons are not on the same row
> image

fixed

**Comment by decentraliser** [comment]
> If I set `_ENABLE_TREZOR_ = true` in the localStorage, I can't access to the Trezor account creation

fixed

**Comment by NemTechCN** [comment]
> in walletChoosen the columns are broken
> image

fixed

**Pull request merged**